exchanges/kucoin/kucoin_websocket.go

```diff
@@ -677,30 +677,27 @@ func (ku *Kucoin) processOrderChangeEvent(respData []byte) error {
 	if err != nil {
 		return err
 	}
-	orderChange := order.Detail{
-		Price:           response.Price,
-		Amount:          response.Size,
-		ExecutedAmount:  response.FilledSize,
-		RemainingAmount: response.RemainSize,
-		Exchange:        ku.Name,
-		OrderID:         response.OrderID,
-		ClientOrderID:   response.ClientOid,
-		Type:            oType,
-		Side:            side,
-		Status:          oStatus,
-		AssetType:       asset.Spot,
-		Date:            response.OrderTime.Time(),
-		LastUpdated:     response.Timestamp.Time(),
-		Pair:            pair,
-	}
-	assetPairEnabled := ku.listOfAssetsCurrencyPairEnabledFor(pair)
-	if assetPairEnabled[asset.Spot] && ku.CurrencyPairs.IsAssetEnabled(asset.Spot) == nil {
-		ku.Websocket.DataHandler <- &orderChange
-	}
-	if assetPairEnabled[asset.Margin] && ku.CurrencyPairs.IsAssetEnabled(asset.Margin) == nil {
-		marginOrderChange := orderChange
-		marginOrderChange.AssetType = asset.Margin
-		ku.Websocket.DataHandler <- &marginOrderChange
+	// should amend this function as we need to know the order type when we call it
```

Comment:
Need to literally write TODO: 
Also, `order type` should be `order asset type`

The crux of the issue here is that I can't see how it could be right to guess asset types for an order based on enabled Assets.

Reply:
comment changed

Reply:
Yeah we going to have to fix this, but not in this PR. 😆